services.yaml

```diff
@@ -304,6 +304,33 @@ paths:
           $ref: "#/responses/TBD"
         401:
             $ref: "#/responses/UnauthorizedError"
+  /services/{service_id}/jvms:
```

Comment:
it can't be JVM as this is too specific. I would at the very least go with "agents".
Also, today the system is coupling an agent to a collector, if we flatten this to a list of agents (which i'm fine with) we need to consider maybe to add a "collectors" endpoint.
both are valid